src/object/set_object.c

```diff
@@ -6153,7 +6154,7 @@ setobj_print (FILE * fp, COL * col)
  */
 
 DB_TYPE
-setobj_type (COL * set)
+setobj_type (struct setobj *set)
```

Comment:
Do you want to change all occurrences of COL or just this?

Reply:
just this one. setobj_type is a special case; it is exposed by dbtype.h to C compiled units. the rest of functions are not.

Reply:
Then please add comments.